.github/workflows/_push_image.yaml

```diff
@@ -0,0 +1,78 @@
+name: Push images
+
+# This is a reusable workflow that is called from any other workflow that needs 
+# to build and push the configurator image to DockerHub
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Version as is going to use for the binary'
+        type: string
+        required: true
+      image-tag:
+        description: 'Tag as is going to use for the image'
+        type: string
+        required: true
```

Comment:
isn't the tag linked to the version? I would not de-couple them unless we need it